### PR TITLE
Add table of examples (#469)

### LINKED ISSUE #469: table of egs
would like to take Fred Richardson's table, see
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kaldi-developers/hUM60Njtu5g/978NyJoDAQAJ
and make it part of the documentation.  need someone familiar with doxygen to make a pull request.

### PR DESCRIPTION
**Pull request by guoguo12**
I converted Fred Richardson's table (posted [here](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kaldi-developers/hUM60Njtu5g/978NyJoDAQAJ)) into an HTML table and made a new documentation page for it.

I didn't do any proofreading/fact-checking, but I'm willing to.

This resolves #469.

### CODE REVIEW

File: src/doc/examples.dox [hunk 1]
```diff
@@ -0,0 +1,362 @@
+// doc/examples.dox
+
+// Copyright 2016 Fred Richardson
```
Comment (danpovey) [inline src/doc/examples.dox]: you could put your name here too.

**Comment by danpovey** [comment]
Thanks..
I'm a little concerned about this being hard to maintain-- the format makes it very hard to understand what field of the table each line of the .dox file corresponds to, so for someone to add a new example to the table - or even to edit an existing one- would be quite hard.  Can you see if you can figure out a way to make it easier?

**Comment by guoguo12** [comment]
Added some comments to make it easier to read.

**Comment by danpovey** [comment]
Thanks.
I think there should be a field for the directory, and there should be a preamble that explains that once you check out the Kaldi repository, the scripts can be found in subdirectories of egs/.  In future, in addition to just the WER we should probably be specifying what type of model it was  (e.g. an nnet1 xent+SMBR model; an nnet2 TDNN; an nnet3 chain+BLSTM+sMBR model, etc.  But this may be hard for you to do as it's time consuming, so for now just add a placeholder field for it.
Dan

**Comment by guoguo12** [comment]
Added a brief preamble and a "Model Type" field.

I think it's pretty clear which subdirectories of `egs/` correspond to which rows in the table. Are there any that are particularly ambiguous?

**Comment by danpovey** [comment]
I compiled this so it appears on the site right now.  @kkm000, would you mind commenting on this?

**Comment by danpovey** [comment]
I'm merging this- I may reword it slightly or move it around, but it's a good start.

**Comment by guoguo12** [comment]
Thanks! Let me know if I can help with anything.

**Pull request merged**